notebooks/official/pipelines/automl_tabular_classification_beans.ipynb

```diff
@@ -2,13 +2,13 @@
   "cells": [
```

Comment:
<p>imports for GCPC should be defined inside of pipeline</p><p> </p><p>gcc_aip.ModelDeployOp( -&gt; _ = gcc_aip.ModelDeployOp(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/787/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
done...